### ISSUE TITLE
Charts out of bounds when not in the Top 2 Bottom direction.

### ISSUE THREAD

**Issue by St1fle** [dabeng/OrgChart#77]
Hello dabeng!

Once again thank you for your help with your request, the company's directors were really happy with the functionality.
Unfortunately I have noticed something that is visually unappealing of which I think it's a bug. When you define a regular chart without specifying the direction the chart is locked in to the div that it is in. For example:

`<div class="row">
    <div class="col-md-12">
        <div id="graph"></div>
    </div>
</div>"`

And if the row is wrapped in a 400/400 width/height div the graph is locked in to that 400/400 div (you can't move the graph outside the div). But if you change the direction to l2r for example, the chart moves outside of the div. You can actually drag it around over any part of the page. I can provide images if necessary.

I thought this might be a bug because if the chart is locked at the default direction then it should be locked in other directions too. I request that this be fixed please!

Thank you very much,
St1fle

**Comment by dabeng** [comment]
Hi @St1fle , if you enable option "pan" then you can drag the chart in any directions. But I don't understand why you just set option "direction" to l2r and you can drag the chart around over the page.

**Comment by St1fle** [comment]
Hello @dabeng,

I'm so sorry for taking so long to answer you. We've had a lot of work in the company and I was unable to explain myself properly. I don't know how to explain this better so I will try to show what my problem is with pictures.

A horizontal layout has this problem (this is what it looks like when we load the page):
![Horizontal Graph](http://i.imgur.com/CqeR9og.jpg)

It forces the user to zoom out many times back and it becomes very small, another problem we have (and we had to disable zoom because of this) is that when we zoomed in the graph we were also scrolling on the page so people were struggling with controlling the zoom, is there something on my side that I can change to fix this?

So we decided to use a vertical layout like this, which looked great by the way:
![VerticalGraph](http://i.imgur.com/PkX9JfN.jpg)

But this vertical layout when you drag it, it isn't "locked" to the chart space, it can be dragged *anywhere* in the page. This is the problem I was trying to point out (notice how in the second picture the chart is over the sidebar on the left, instead of dragging around the chart space).

Horizontal cuts user davideferreira (this is what we want):
![VerticalGraph](http://i.imgur.com/13uXzYK.jpg)

But vertical layout overflows anywhere on the screen:
![VerticalGraph](http://i.imgur.com/cYlPVQW.jpg)

I hope this explains my situation better and I am sorry for not being very explicative in the first place.
